deployment/geonode/templates/geonode/geonode-env.yaml

```diff
@@ -35,7 +35,6 @@ data:
 
   GEONODE_INSTANCE_NAME: {{ .Release.Name }}
   GEONODE_LB_HOST_IP: {{ .Values.geonode.ingress.externalDomain | quote }}
-  GEONODE_LB_PORT: {{ .Values.geonode.ingress.externalPort | quote }}
```

Comment:
GEONODE_LB_PORT must be set as there is no default value defined inside the tasks.py overlay. To better debug geonode container initialization i removed most of the default values while loading environment variables in tasks.py. The environment variables should either be loaded or error out directly at the variable loading.

Reply:
Could you link the location where you get that error exactly?

Reply:
Could you re-check my comment before? I did an edit on it.